### ISSUE TITLE
Upload speed throttling affects local file backups

### ISSUE THREAD

**Issue by mkatsevVR** [duplicati/duplicati#2685]
I have:
- [x] searched open and closed issues for duplicates

----------------------------------------

### Version info
**Duplicati Version:** 2.0.1.73_experimental_2017-07-15
**Operating System:** Windows 10
**Backend:** Local folder/drive

### Bug description
Enabling global upload throttling in the web ui status bar applies to local backups as well

### Steps to reproduce
- Create a local backup
- Enable throttling
- Set upload speed to various values
- Monitor actual write speeds with Task Manager

**Actual result:** Write speed to the drive follows the upload limit exactly
**Expected result:** Upload limit should not apply to local drives (or at the very least be a separate setting)

**Comment by shoeper** [comment]
Could be interesting to allow write speed limitation, though. It could prevent the backups occupying to many resources.

**Comment by mkatsevVR** [comment]
i agree it's reasonable to have a configurable limit, but in most cases it would be a much higher value than what you'd use for network uploads
I think the best UX would be to ignore the global throttle for local jobs, but still respect the job specific one, but perhaps it would be difficult to implement depending on how the global throttle works under the hood?

**Comment by JonMikelV** [comment]
With the recent addition of the `--use-background-io-priority` Advanced option, I wonder if the potential use of a throttle on local speed no longer would apply.

**Comment by ts678** [comment]
A use case where throttling seemingly local folder backups is good is when the folder is actually a network folder. Example [here](https://forum.duplicati.com/t/limit-the-size-of-temporary-files-stored-in-local-disk/11353).

**Comment by ts678** [comment]
and it's also useful if one wants to actually set the speed, instead of taking whatever `--use-background-io-priority` provides, which on a busy system can be nothing (but it's what was requested). Seeing nobody else asking for this in six years makes me wonder if there's much of an actual problem here that can't be worked out with a nice combination of global and local throttles.

Re: issue's title strictly, I see "Upload speed throttling affects local file backups" as the right plan. Don't remove ability to throttle.
